airbyte-ci/connectors/connector_ops/tests/test_required_reviewer_checks.py

```diff
@@ -23,7 +23,7 @@ def mock_diffed_branched(mocker):
 
 @pytest.fixture
 def pokeapi_metadata_path():
-    return "airbyte-integrations/connectors/source-pokeapi/metadata.yaml"
+    return "airbyte-integrations/connectors/source-zoho-crm/metadata.yaml"
```

Comment:
Updated as source Pokeapi is now manifest-only, which breaks a couple test cases for expected reviewers. Zoho CRM is written in Python, which makes it unlikely to be migrated to manifest-only anytime soon.